document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/data/transaction/Resources.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.companieshouse.document.generator.accounts.data.transaction;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import java.util.Date;
+import java.util.Map;
+
+public class Resources {
+
+    @Field("updated_by")
```

Comment:
By using `@Field`, we're tying ourselves in to using a `MongoDB` dependency through `Spring Boot`... Do we NEED this? Can it be replaced with something else?

Reply:
Removed and tested, doesn't look like its necessary for what we are doing.